### DISCUSSION TITLE
How can I use images in relative path?

### DISCUSSION THREAD

**Discussion by TetraTheta** [hbstack/theme#380] [Q&A] [answered]
I prefer putting images and markdown file in same parent directory like this:
```
content/game/minecraft/
├ 001
│ ├ 001.png
│ ├ 002.png
│ ├ 003.png
│ ├ 004.png
└ 001-todays-construction.md
```
But when I tried to use those images in markdown file, they don't show up. (testing in live preview mode)

I've tried with these syntaxs:
```
![](001/001.png)
![](./001/001.png)
![](/001/001.png)
![](../001/001.png)
```
None of these works and Hugo says `WARN  image not found: ./001/001.png`.

How can I use images in my post?

**Comment by razonyang** [comment, marked as answer]
Hi, it won't work with your current content structure. In short, you should put the images under the _page_ folder.

```
content/game/minecraft/
├ 001-todays-construction
│ ├ index.md
│ ├ 001.png
│ ├ 002.png
│ ├ 003.png
│ ├ 004.png
```

The `index.md` makes the page (`/game/minecraft/001-todays-construction/`) become a leaf bundle, if you want to share the images cross multiple pages, you can either save it in `assets` (URL should begin with a leading slash `/`) or `static` folder.

Read more on:

- https://hugomods.com/en/docs/images/#image-types
- https://gohugo.io/content-management/page-resources/
- https://gohugo.io/content-management/page-bundles/

**Reply by TetraTheta** [reply]
Thank you for the fast reply! Now the images are showing correctly.

**Reply by razonyang** [reply]
You're welcome.
